mnist/README.md

```diff
@@ -1,170 +1,99 @@
+    --name=minio -d --net=host -v /var/lib/minio:/data minio/minio server /data
 ```
 
-## Modifying existing examples
-
-Many examples online use models that are unconfigurable, or don't work well in distributed mode. We will modify one of these [examples](https://github.com/tensorflow/tensorflow/blob/9a24e8acfcd8c9046e1abaac9dbf5e146186f4c2/tensorflow/examples/learn/mnist.py) to be better suited for distributed training and model serving.
-
-### Prepare model
-
-There is a delta between existing distributed mnist examples and what's needed to run well as a TFJob.
-
-Basically, we must:
+The Minio access key is `minio`, secret key is `minio123`. We will use them in later steps.
 
-1. Add options in order to make the model configurable.
-1. Use `tf.estimator.train_and_evaluate` to enable model exporting and serving.
-1. Define serving signatures for model serving.
 
-The resulting model is [model.py](model.py).
-
-### Build and push model image.
-
-With our code ready, we will now build/push the docker image.
+### Create Minio bucket
 
 ```
-DOCKER_BASE_URL=docker.io/elsonrodriguez # Put your docker registry here
-docker build . --no-cache  -f Dockerfile.model -t ${DOCKER_BASE_URL}/mytfmodel:1.7
-
-docker push ${DOCKER_BASE_URL}/mytfmodel:1.7
+mkdir /var/lib/minio/tfmnist
 ```
 
-## Preparing your Kubernetes Cluster
-
-With our data and workloads ready, now the cluster must be prepared. We will be deploying the TF Operator, and Argo to help manage our training job.
-
-In the following instructions we will install our required components to a single namespace.  For these instructions we will assume the chosen namespace is `tfworkflow`:
-
-### Deploying Tensorflow Operator and Argo.
 
-We are using the Tensorflow operator to automate the deployment of our distributed model training, and Argo to create the overall training pipeline. The easiest way to install these components on your Kubernetes cluster is by using Kubeflow's ksonnet prototypes.
+## 5.Download argo
 
 ```
-NAMESPACE=tfworkflow
-APP_NAME=my-kubeflow
-ks init ${APP_NAME}
-cd ${APP_NAME}
-
-ks registry add kubeflow github.com/kubeflow/kubeflow/tree/v0.2.4/kubeflow
-
-ks pkg install kubeflow/core@v0.2.4
-ks pkg install kubeflow/argo
-
-# Deploy TF Operator and Argo
-kubectl create namespace ${NAMESPACE}
-ks generate core kubeflow-core --name=kubeflow-core --namespace=${NAMESPACE}
-ks generate argo kubeflow-argo --name=kubeflow-argo --namespace=${NAMESPACE}
-
-ks apply default -c kubeflow-core
-ks apply default -c kubeflow-argo
-
-# Switch context for the rest of the example
-kubectl config set-context $(kubectl config current-context) --namespace=${NAMESPACE}
-cd -
-
-# Create a user for our workflow
-kubectl apply -f tf-user.yaml
+curl -sSL -o /usr/local/bin/argo https://github.com/argoproj/argo/releases/download/v2.2.1/argo-linux-amd64
+chmod +x /usr/local/bin/argo
 ```
 
-You can check to make sure the components have deployed:
+## 6.Create tf service account
```

Comment:
What is this service account for?

Reply:
added more description about this serviceaccount.

Reply:
Did you forget to add tf-user.yaml to the PR?

Reply:
it is already in the example, I did not change it, so there is not diff in this PR. https://github.com/chenzhiwei/kubeflow-examples/blob/mnist-model/mnist/tf-user.yaml